gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -793,6 +910,73 @@ public static LicenseOption fields(LicenseField... fields) {
   }
 
   /**
+   * Class for specifying operation get options.
+   */
+  class OperationOption extends Option {
```

Comment:
Almost all Compute APIs (but gets) are async and return an operation. Field selection for operations can be provided in all such APIs, calling the option `OperationOption` sounded better to me given that it must be passed to methods that have nothing to do with `getOperation`.

And you are right in storage we have so many blob-related options that it's worth clarifying.